src/overlays/actors/ovl_En_Fall/z_en_fall.c

```diff
@@ -24,6 +24,9 @@
 
```

Comment:
Since you name the functions by the subtypes, I think it would be a good idea to include the names you use in the file description at the top; basically make it a description list rather than an unordered one, to use the html terms.